docs/sources/enterprise/usage-insights/dashboard-datasource-insights.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Dashboard and data source insights"
+description = "Understand how your dashboards and data sources are used"
+keywords = ["grafana", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/dashboard-datasource-insights.md"]
+weight = 200
++++
+
+# Dashboard and data source insights
+
+For every dashboard and data source, you can access usage information in one click.
+
+## Dashboard insights
+
+> **Note:** Available in Grafana Enterprise v7.0+.
+
+You can see dashboard usage information by clicking on the `Dashboard insights` button in the top bar.
+
+{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_button.png" max-width="400px" class="docs-image--no-shadow" >}}
+
+It shows two kinds of information:
+
+- **Stats:** Shows the daily query count and error count for the last 30 days.
+- **Users & activity:** Shows the daily view count for the last 30 days; last activities on the dashboard and recent users (with a limit of 20).
+
+{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_stats.png" max-width="400px" class="docs-image--no-shadow" >}}{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_users.png" max-width="400px" class="docs-image--no-shadow" >}}
+
+## Data source insights
+
+> **Note:** Available in Grafana Enterprise v7.3+.
+
+Data source insights give you information about how a data source has been used in the last thirty days.
+
+- Queries per day
```

Comment:
```suggestion
- Queries
```

Reply:
What about `Query count` or `Number of queries`? Writing only `Queries` let me think that I could access all the queries that were made in the past 30 days.
You can see the actual display here: https://grafana.com/static/img/docs/enterprise/datasource_insights.png